backend/src/controllers/groups-controller.ts

```diff
@@ -0,0 +1,106 @@
+import { Request, Response } from 'express';
+import Group from '../models/group';
+import { validationResult } from 'express-validator';
+import group from "../models/group";
```

Comment:
Group was imported twice, once as `Group` and once as `group`. Remove the `group` import to reduce repetitiveness and since it can be confused with definitions of `const group = ...

Reply:
Replace all instances where this `group` is used (e.x. `group.____`) with `Group.____`